pages/manage-contributions.js

```diff
@@ -100,6 +103,12 @@ class ManageContributionsPage extends React.Component {
     const adminMemberships = loggedInUser?.memberOf?.filter(m => m.role === 'ADMIN' && !m.collective.isIncognito);
     const uniqMemberships = uniqBy(adminMemberships, 'collective.id');
     const groupedMemberships = groupBy(uniqMemberships, 'collective.type');
+    const projectMemberships = concat(...uniqMemberships.map(m => m.collective.children)).filter(
+      m => m.type === CollectiveType.PROJECT,
+    );
+    if (projectMemberships.length > 0) {
+      groupedMemberships[CollectiveType.PROJECT] = projectMemberships;
+    }
     return mapValues(groupedMemberships, memberships => orderBy(memberships, 'collective.name'));
```

Comment:
Because we don't need to filter out other profile types, and because we don't really need the membership (only the `collective`), we can simplify the code and remove the `memberCollective` function that makes the code more difficult to read. Plus, this function is called `getAdministratedAccounts`, it makes sense to return a list of accounts rather than a list of memberships.

```suggestion
    const adminAccounts = adminMemberships.map(m => m.collective);
    const childrenAdminAccounts = flatten(adminAccounts.map(c => c.children));
    const uniqAccounts = uniqBy([...adminAccounts, ...childrenAdminAccounts], 'id');
    const groupedAccounts = groupBy(uniqAccounts, 'type');
    return mapValues(groupedAccounts, accounts => orderBy(accounts, 'name'));
```

Reply:
@Betree : Done. Thanks for suggesting this.